monai/losses/dice.py

```diff
@@ -778,12 +778,22 @@ def forward(self, input: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
 
         Raises:
             ValueError: When number of dimensions for input and target are different.
-            ValueError: When number of channels for target is neither 1 nor the same as input.
+            ValueError: When the target channel isn't either one-hot encoded or categorical with the same shape of the input.
```

Comment:
The error message seems to indicate that the data must either be in one-hot format or consistent with the original image, which appears to slightly differ from the initial implication.

Reply:
Yes, sorry, on re-reading the original message is clearer, I've changed that, hoping it's better now!